packages/calypso-e2e/src/jest-playwright-config/playwright-config.ts

```diff
@@ -12,6 +12,8 @@ const getTargetDeviceOptions = () => {
 		case 'desktop':
 			deviceName = 'Desktop Chrome HiDPI';
 			break;
+		default:
+			throw new Error( 'No viewport specified.' );
```

Comment:
Not sure how this didn't throw up compilation error prior to this PR. In any case, moving the `VIEWPORT_NAME` out of `envVariables` started to trigger that a default case is not present.